json/src/v28/raw_transactions.rs

```diff
@@ -0,0 +1,112 @@
+// SPDX-License-Identifier: CC0-1.0
+
+//! The JSON-RPC API for Bitcoin Core v28.0 - raw transactions.
+//!
+//! Types for methods found under the `== Rawtransactions ==` section of the API docs.
+
+use std::collections::HashMap;
+
+use bitcoin::{Amount, FeeRate, Txid, Wtxid};
+use serde::{Deserialize, Serialize};
+
+/// Models the result of JSON-RPC method `submitpackage`.
```

Comment:
```suggestion
/// Result of JSON-RPC method `submitpackage`.
```

So folk don't get confused between this and the `model` module types.